rps.js

```diff
@@ -0,0 +1,48 @@
+
+let cmChoice = Math.random();
+
+const game = function(user){
+if (cmChoice < 0.33) {
+    cmChoice = "rock";
+}
+else if( cmChoice > 0.66) { 
+    cmChoice = "scissors";
+}
+else {
+    cmChoice = "paper"
+}
+if (playGame) {
+    let user = "rock"
+} return cmChoice
+}
+const choices = function(cmChoice, user) {

```

Comment:
you can write  the entire code in the same function you made previously `game`, you  don't need to make a new function.